packages/workflow/src/node-parameters/parameter-type-validation.ts

```diff
@@ -0,0 +1,253 @@
+import { NodeOperationError } from '../errors';
```

Comment:
moved these utilities to workflow package so they can be reused across node packages